src/main/java/io/github/acm19/aws/interceptor/http/AwsRequestSigningApacheInterceptor.java

```diff
@@ -146,7 +147,8 @@ public void process(final HttpRequest request, final HttpContext context)
                 basicHttpEntity.setContent(signedRequest.contentStreamProvider()
                         .orElseThrow(() -> new IllegalStateException("There must be content"))
                         .newStream());
-                httpEntityEnclosingRequest.setEntity(basicHttpEntity);
+                // wrap into a repeatable entity, see https://github.com/acm19/aws-request-signing-apache-interceptor/issues/38
```

Comment:
I'd replace the link to the issue by a short explanation to make the code easier to read. Something like:
> wrap into repeatable entity to support retries